client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-card/index.tsx

```diff
@@ -180,7 +180,7 @@ export default function HostingCard( {
 					{ highestDiscountPercentage ? (
 						<div className="hosting-card__price-discount">
 							{ translate( 'Volume savings up to %(highestDiscountPercentage)s%', {
-								args: { highestDiscountPercentage },
+								args: { highestDiscountPercentage: Math.trunc( highestDiscountPercentage ) },
```

Comment:
This looks good enough for now, but I think we should include a TODO message to revisit this later with a different approach.

Reply:
I don't get it. Could you elaborate the issue a bit more?

Reply:
To make sure all percentages are displayed the same way across A4A, we could use a single function for formatting. This can be done with a utility function, which keeps the formatting consistent.

But using `Math.trunc` is pretty straightforward and effective for our current needs.

Reply:
> using Math.trunc is pretty straightforward and effective for our current needs.

^^This. As you say, we already have that utility function for it; it's just to use `Math.trunc`.